common/src/main/scala/com/gu/recipeasy/models/CuratedRecipe.scala

```diff
@@ -85,27 +85,47 @@ case class TimesInMins(
 
 sealed trait CookingUnit {
   def abbreviation: String
+  def displayName: String
 }
 
 object CookingUnit {
   val unitMap = Map(
+    "cup" -> Cup,
```

Comment:
The key in this map corresponds to the abbreviation on the corresponding object, correct? If so, we should reference Cup.abbreviation as the key here so we don't have it in two separate places.